test/src/Makefile

```
@@ -0,0 +1,50 @@
+PACKAGE  = terraform-aws-ecs-container-definition
+GOEXE   ?= /usr/bin/go
+GOPATH   = $(CURDIR)/.gopath
+GOBIN    = $(GOPATH)/bin
+BASE     = $(GOPATH)/src/$(PACKAGE)
+PATH 	:= $(PATH):$(GOBIN)
+
+export TF_DATA_DIR ?= $(CURDIR)/.terraform
+export TF_CLI_ARGS_init ?= -get-plugins=true
+export GOPATH
+
+.PHONY: all
+## Default target
+all: test
+
+ifneq (,$(wildcard /sbin/apk))
+## Install go, if not installed
+$(GOEXE):
+	apk add --update go
+endif
+
+ifeq ($(shell uname -s),Linux)
```

@Gowiem recommends updating this line. I updated the same in a different repository and it found my `go` version correctly.

```suggestion
ifeq ($(shell uname -s),$(filter $(shell uname -s),Linux Darwin))
```

---

Good call. I've reworked it a little bit, but accepted the suggestion.